test/gtest/uct/ib/test_rc.cc

```diff
@@ -52,6 +52,8 @@ class test_rc_max_wr : public test_rc {
 /* Check that max_wr stops from sending */
 UCS_TEST_P(test_rc_max_wr, send_limit)
 {
+    check_caps(UCT_IFACE_FLAG_AM_SHORT);
```

Comment:
i see many tests would be skipped now if no inline send.. i think it's better to switch these tests to bcopy rather than disabling them (unless bcopy variant is already tested in same test case and then it's ok to skip)